src/test/java/org/jabref/logic/pdf/PdfAnnotationImporterTest.java

```diff
@@ -0,0 +1,130 @@
+package org.jabref.logic.pdf;
+
+import java.nio.file.Paths;
+import java.util.List;
+
+import org.jabref.model.pdf.FileAnnotation;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class PdfAnnotationImporterTest {
+
+    private AnnotationImporter importer = new PdfAnnotationImporter();
+
+    @Test
+    public void noAnnotationsWriteProtected() {
+
+        List<FileAnnotation> annotations = importer.importAnnotations(Paths.get("src/test/resources/pdfs/write-protected.pdf"));
+        assertEquals(0, annotations.size());
+    }
+
+    @Test
+    public void noAnnotationsEncrypted() {
+        List<FileAnnotation> annotations = importer.importAnnotations(Paths.get("src/test/resources/pdfs/encrypted.pdf"));
+        assertEquals(0, annotations.size());
+    }
+
+    @Test
+    public void twoAnnotationsThesisExample() {
+
+        List<FileAnnotation> annotations = importer.importAnnotations(Paths.get("src/test/resources/pdfs/thesis-example.pdf"));
+        assertEquals(2, annotations.size());
+    }
+
+    @Test
+    public void noAnnotationsMinimal() {
+        List<FileAnnotation> annotations = importer.importAnnotations(Paths.get("src/test/resources/pdfs/minimal.pdf"));
+        assertEquals(0, annotations.size());
+    }
+
+    @Test
+    public void inlineNoteMinimal() {
+        List<FileAnnotation> annotations = importer.importAnnotations(Paths.get("src/test/resources/pdfs/minimal-inlinenote.pdf"));
+        assertEquals(1, annotations.size());
```

Comment:
Hm. After @tobiasdiez comment I have tried it already but the message is: (see 8ed6abf1a9a3a486f7d62b84e153b1a14130c18d)
```
java.lang.AssertionError: Comparison fails. WHY? expected: java.util.Collections$SingletonList<[inline note annotation]> but was: java.util.LinkedList<[inline note annotation]>
Expected :java.util.Collections$SingletonList<[inline note annotation]> 
Actual   :java.util.LinkedList<[inline note annotation]>
```
First of all I don't get why the comparison fails.
Second the message is really not helpful.